### ISSUE TITLE
/mailings query filter 'updated_since' always returns 400 Bad Request

### ISSUE THREAD

**Issue by ghost** [rapidmail/rapidmail-apiv3-client-php#2]
The [documentation](https://developer.rapidmail.wiki/documentation.html?urls.primaryName=Mailings#/Mailings/get_mailings) says `updated_since` can be set to a DateTime in the format `Y-m-d H:i:s`. This will be transmitted as a URL-encoded GET-parameter, like `updated_since=2020-04-08%2016%3A46%3A54` for the DateTime `2020-04-08 16:46:54`. This results in an API error response 400 Bad Request with the message:

    API error 400 when requesting https://apiv3.emailsys.net/v1/mailings?updated_since=2020-04-01. Detail: {"validation_messages":{"updated_since":{"datetimeInvalidDateTime":"The input does not appear to be a valid datetime"}},"type":"http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html","title":"Bad Request","status":400,"detail":"Failed Validation"}

Any way I try to format the date results in the same error. UNIX epoch timestamp, `2020-04-08`, nothing seems to be acceepted.

As an aside: the code https://github.com/rapidmail/rapidmail-apiv3-client-php/blob/3081ac10e03517a7f1a77d3198245d963facdc1c/src/Service/V1/Api/Mailings/Mailing/Parameter/MailingQueryParam.php#L100 sets the wrong GET parameter.

**Comment by eth8505** [comment]
Thank you for the repost @faltrock-abone, there seem to be two problems here.

1. The setter is indeed populating the wrong field
2. The date format specified in the docs is currently wrong. The system should accept the format "Y-m-d H:i:s", but currently expects the format "Y-m-d\TH:i:s". As soon as we get this fixed, it will accept both, with the preferred (and documented) format still being "Y-m-d H:i:s". 

I created issues for these in our internal issue tracker and we will try to get this fixed asap.

**Comment by eth8505** [comment]
@faltrock-abone I just published version 1.0.1 of the API client that should resolve the issue with the wrong field being set. I will try to get the date parsing fixed in the API as soon as I can.

In the mean time, you could simply set the date in the currently expected format "Y-m-d\TH:i:s" like this:
```php
$client->mailings()->query([
    'updated_since' => '2020-01-01T10:00:00'
]);
```

When we get the API fixed the default expected (and documented) date format will be "Y-m-d H:i:s", but as previously stated we will also be supporting "Y-m-d\TH:i:s" for backwards compatibiilty.

**Comment by eth8505** [comment]
@faltrock-abone the API should also be fixed now supporting the date format "Y-m-d H:i:s" for the GET /mailings filter fields.
